README.md

```diff
@@ -730,3 +731,110 @@ git commit
 git push origin develop
 ```
 
+## Updating a production instance
+
+Every now and then, the production instance needs to be updated, so the server
+can get the latest security patches, and the Research Software Directory
+software itself can be updated to include the latest features.
+
+The steps below differentiate between the old and the new instance of the Research
+Software Directory; the old instance has IP ``35.156.38.208``, the new one has
+IP ``3.122.233.225``. Your IP addresses will likely be different.
+
+1. Make a new Amazon instance by following the notes above. Some things to think about:
+    - Reuse the existing security group.
+    - Reuse the existing key pair.
+    - Verify that your current IP is within the authorized IP range.
+1. Download the ``rsd-secrets.env`` file from the old instance.
+
+    ```
+    # something like:
+    $ cd $(mktemp -d)
+
+    $ scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem \
+      ubuntu@35.156.38.208:/home/ubuntu/rsd/rsd-secrets.env .
+    ```
+
+1. Upload ``rsd-secrets.env`` to the new Research Software Directory
+   instance.
+
+    ```
+    $ scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem rsd-secrets.env \
+    ubuntu@3.122.233.225:/home/ubuntu/rsd/
+    ```
+
+1. Stop new additions to the database in the old research software
```

Comment:
Also copy `docker-volumes/cert` and `docker-volumes/letsencrypt`.
Otherwise you will get a self-signed cert and visitors will get errors.